tests/integration/conftest.py

```diff
@@ -282,63 +316,56 @@ def pod_db_password_fixture():
     return "wordpress-password"
 
 
-@pytest_asyncio.fixture(scope="module", name="build_and_deploy")
-async def build_and_deploy_fixture(
-    request,
-    ops_test: pytest_operator.plugin.OpsTest,
-    application_name,
-    kube_core_client,
-    pod_db_database,
-    pod_db_user,
-    pod_db_password,
+@pytest.fixture(scope="module", name="deploy_and_wait_for_mysql_pod")
+def deploy_and_wait_for_mysql_pod_fixture(
+    ops_test, kube_core_client, pod_db_database, pod_db_user, pod_db_password
 ):
-    """Deploy all required charms and kubernetes pods for tests."""
-    assert ops_test.model
-    num_units = request.config.getoption("--num-units")
+    """Return an async function that deploy and wait for a mysql pod ready in current namespace.
 
-    async def build_and_deploy_wordpress():
-        my_charm = await ops_test.build_charm(".")
-        await ops_test.model.deploy(
-            my_charm,
-            resources={"wordpress-image": "localhost:32000/wordpress:test"},
-            application_name=application_name,
-            series="jammy",
-            num_units=num_units,
-        )
+    This is used for testing WordPress charm's capability of interacting with an external non-charm
+    MySQL database.
+    """
 
-    # create a pod to test the capability of the WordPress charm to interactive with an external
```

Comment:
Do you think this comment is still relevant?

Reply:
This is also caused by merging the `sidecar` branch into this PR. I will add this back too.

Reply:
Added. thanks!